### DISCUSSION TITLE
Select Timing Point by clicking on it in timeline

### DISCUSSION THREAD

**Discussion by kongehund** [ppy/osu#31177] [Ideas]
The "Select closest to current time" option is significantly less intuitive than just clicking on the timing point in the timeline that you're interested in.

https://github.com/user-attachments/assets/92c73920-813b-4cfd-8be7-3f7c7a11fb4a

**Comment by peppy** [comment]
This sounds like a good UX improvement, but we'll need to make `SelectedGroup` more globally accessible for the selection part to work (maybe it should be moved to `EditorBeatmap`?).
